PKGBUILD

```diff
@@ -17,7 +17,6 @@ makedepends=(
   pahole
   perl
   python
-  rustup
```

Comment:
Just removing ` rust-toolchain` from `makedepends` is for sure not the way to go. We should require `rust` and `rust-src` which are provided by either installing these packages or by having `rustup` installed, which provides these.

Reply:
Makes a lot of sense. I never checked what rustup actually provides. 
I have modified it to depend and rust and rust-src, and automatically use rustup if it is available. What do you think about that version?